adminSiteServer/apiRouter.ts

```diff
@@ -497,15 +495,13 @@ apiRouter.get(
     }
 )
 
-apiRouter.get("/topics.json", async (req: Request, res: Response) => ({
+apiRouter.get("/topics.json", async (req, res) => ({
     topics: await DEPRECATEDgetTopics(),
 }))
-apiRouter.get(
-    "/editorData/variables.json",
-    async (req: Request, res: Response) => {
-        const datasets = []
-        const rows = await db.queryMysql(
-            `SELECT
+apiRouter.get("/editorData/variables.json", async (req, res) => {
```

Comment:
This one should be relatively easy to convert to knex, if you want to tackle that one.

Reply:
Jup, true, added this